.github/workflows/download_pipeline.yml

```diff
@@ -76,7 +73,7 @@ jobs:
         env:
           NXF_SINGULARITY_CACHEDIR: ./
           NXF_SINGULARITY_HOME_MOUNT: true
-        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ env.REPO_BRANCH }}) -stub -profile test,singularity --outdir ./results
+        run: nextflow run ./${{ env.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< dev) -stub -profile test,singularity --outdir ./results
```

Comment:
This change is not needed.

Reply:
@Midnighter @sofstam Can we ignore the failing donwlaod-piepline.yml test in #469 ?

Reply:
Fine for me to skip some formalities when upgrading infrastructure.